ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/teaser/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,352 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@Rajamouli-hcl : what are the changes in dialog as you have copied from core here??

Reply:
added node for pretitle which is not prasent core component

Reply:
@Rajamouli-hcl : why do u want to copy yje whole dilaog when u can overy. and add single node. Ref below:
![image](https://user-images.githubusercontent.com/38690824/81469363-1fd71280-9202-11ea-94d4-8236aa4ebcd5.png)

Reply:
i can't do overlay for core components that's way i copied and used the whole dialog
 as per your reference  i checked its working i will do these changes

Reply:
changes has been commited as per the review comments